recipe/meta.yaml

```diff
@@ -1,33 +1,62 @@
+{% set version = "1.2.7" %}
+
+{% if cuda_compiler_version == "None" %}
+{% set cuda_major = 0 %}
+{% else %}
+{% set cuda_major = environ.get("cuda_compiler_version", "11.8").split(".")[0] | int %}
+{% endif %}
+
 package:
   name: catboost
-  version: "1.2.7"
+  version: {{ version }}
+
+source:
+  url: https://github.com/catboost/catboost/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 099ff437993621b86565f3b25150071bdc20623ff77b75ed4d0cf6a37263e1fd
+  patches:
+    - conda.diff
 
 build:
-  number: 0
+  number: 1
   #  Yandex supplies whl files on PyPI for:
-  # - Linux: Python 3.7, 3.8, 3.9, 3.10, 3.11, 3.12
-  # - OS X: Python 3.7, 3.8, 3.9, 3.10, 3.11, 3.12
   # - Win-64: Python 3.7, 3.8, 3.9, 3.10, 3.11, 3.12
   skip: True  # [win32]
   skip: True  # [python_impl == 'pypy']
-  # There are issues with python 3.8 on macOS 11.x: https://github.com/conda-forge/python-feedstock/issues/445
-  #  so Conda has issues with tags compatibility with platform tags 'macosx_11_0_...'
-  #  so disable this configuration for now
-  skip: True  # [osx and x86_64 and (py == 38)]
   # TODO: support Linux on ppc64le: https://github.com/catboost/catboost/issues/2145
   skip: True  # [ppc64le]
+  skip: True  # [linux and cuda_compiler_version == "None"]
+  skip: True  # [win and cuda_compiler_version != "None"]
```

Comment:
Yes, but since we are re-packaging binaries on windows, we don't need a cuda build.

Reply:
> But if I understand the conda logic correctly this is interpreted as 'CUDA not supported'.

No, it's not. conda does not care about whether the re-packaged package had CUDA support or not. `conda-build` creates builds for CUDA and no CUDA, but there's no way for us to tell conda-build that we don't need separate builds for windows only.

Also, `cuda_compilerNone` is the default even with the previous logic. It's implicit because we didn't use that variable and it has become explicit here.